### DISCUSSION TITLE
Feature request: enabling the CLI to open the manager screen.

### DISCUSSION THREAD

**Discussion by TheHippoMan** [mywalkb/LSPosed_mod#14] [Ideas]
Now that we have the CLI, I think it would be desirable for it to accept an extra command such as "manager" or something similar which would open the LSPosed_mod manager screen. Then, we could use that command to avoid the three "parasitic" methods for opening this manager screen (the parasitic shortcut, the status notification, or the USSD dialer sequence).

Much can be done now via the CLI command, but there are a still few things that are more easily configured via the manager screen. Allowing us to open the manager screen directly via a CLI command in addition to doing so via one of the "parasitic" methods would therefore be a useful convenience.

**Comment by mywalkb** [comment]
CLI is not enabled by default, so the first time require the start from notification.
I don't know how many people use CLI, I have no a feedback.
We could enable by default and insert opening manager from cli.
In this period I have no spare time.

**Reply by TheHippoMan** [reply]
I understand that you have no spare time. I don't mean to give you any pressure.

And I understand that the parasitic invocation is needed initially. But is it possible some time in the future to add the following command to the CLI?

**_/path/to/cli manager_**

Then, after the CLI has been initially enabled via the parasitic manager, that command could be utilized in order to cause the LSPosed manager window to open?

Then, those of use who use the CLI would only need to use the status notification or the parasitic shortcut once, initially, and then after that, the LSPosed manager could be invoked via that proposed command?

**Reply by mywalkb** [reply]
yes is possible, but you can already execute a cli command for start parasitic manager, also in original LSPosed
as user root execute:

`am start -c org.lsposed.manager.LAUNCH_MANAGER com.android.shell/.BugreportWarningActivity
`

**Reply by TheHippoMan** [reply]
Oh! I didn't now about that LAUNCH_MANAGER command. That's very helpful. Thank you!

**Comment by TheHippoMan** [comment]
Thank you for your help! I'm closing this feature request now.
